lib/steps.js

```diff
@@ -66,6 +66,8 @@ let click_with = {
   pc: 'click',
 };
 
+scope.row_needs_trigger_flag = `ALKiln: no trigger variable needed`;
```

Comment:
We may have mentioned this before, but what's the reason that this string isn't just defined inside `scope.js`? You still have the problem on needing to define it in each file that uses scope (steps and the unit test).

Reply:
It was partly because I thought it was being used in steps.js (which it wasn't) and partly because I was instantiating it in the script where other state variables were being defined. I haven't defined any `scope` props in scope.js other than functions, but I can switch it to there.